README.md

```diff
@@ -17,28 +17,30 @@ Development builds are also available from [our website](https://pcsx2.net/downl
 
 ## Minimum
 * OS: Windows Vista SP2 or newer or GNU/Linux (32-bit or 64-bit)
-* CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up)
+* CPU: Any that supports SSE2 (Pentium 4 and up, Athlon64 and up) @ 1600 STR or better
```

Comment:
Most of the common users won't even be aware of the term ``STR`` and the value you're referencing here is from testcases on passmark single thread benchmarks and you don't seem to have any mentions of that either. I feel this would just look cryptic for any random user.